README.md

```diff
@@ -1,2 +1,39 @@
-# Web3Studio JS Standards
+# Web3Studio ESlint Config
+
 A collection of boilerplate config to be used by Web3Studio javascript based projects.
+
+## Installation
+
+```bash
+$ yarn add --dev eslint-config-web3studio eslint prettier pretty-quick husky
```

Comment:
I included husky and pretty quick b/c this is really the js-standards repo... eslint is just picky about names 🤷‍♂️

Reply:
clarification question - you weren't able to call the repo js-standards or web3studio-js-standards?

Reply:
Correct, for eslint to work normally, it needs to be eslint-(config|plugin)-*